shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/LogInViewModel.kt

```diff
@@ -0,0 +1,66 @@
+package co.nimblehq.avishek.phong.kmmic.presentation.module
+
+import co.nimblehq.avishek.phong.kmmic.data.remote.helper.extension.toErrorMessage
+import co.nimblehq.avishek.phong.kmmic.domain.usecase.LogInUseCase
+import co.nimblehq.avishek.phong.kmmic.helper.extension.isValidEmail
+import kotlinx.coroutines.flow.*
+
+data class LogInViewState(
+    val isSuccess: Boolean = false,
+    val isLoading: Boolean = false,
+    val error: String? = null,
+    val isInvalidEmail: Boolean = false,
+    var isInvalidPassword: Boolean = false
+) {
+    constructor() : this(false)
+
+    constructor(error: String?) : this(false, false, error)
+}
```

Comment:
We don't need the constructors here.
```suggestion
)
```

Reply:
@kaungkhantsoe, iOS module is using those constructor, please check the file `LogInCombineViewModel`